### PR TITLE
Add unit test for the actions of feed, msg, plugin, ui, and user

### PR DESCRIPTION
**Pull request by ethanhou99**
This is a combination work of Yicun, Haoyu, and Qingxing

### CODE REVIEW

File: __test__/store/feed/action.test.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { FeedActionTypes,  } from "../../../src/store/feed/types";
+import { IFeedItem } from "../../../src/api/models/feed.model";
+import { IPluginItem } from "../../../src/api/models/pluginInstance.model";
+import * as actions from "../../../src/store/feed/actions"; 
+
+describe('actions', () => {
```
[the remark this answers is not in the document]
Comment (ethanhou99) [reply __test__/store/feed/action.test.ts]: The feed actions and tested functions are declared.

File: README.md [hunk 2]
```diff
@@ -62,6 +62,43 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
+## Unit Test
+
+The unit test scripts are under `./__test__` folder and tested functions are under `./src/store`.
```
Comment (jcaianirh) [inline README.md]: can we call this __tests__  please?

File: README.md [hunk 3]
```diff
@@ -62,6 +62,43 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+
+The tested functions are:
```
Comment (jcaianirh) [inline README.md]: no need to list every test, just add a summary of whats being tested.

**Comment by jcaianirh** [comment]
When I run '''npm test''' and run all tests I get the error: 
 Cannot find module './App' from 'App.test.tsx'
@rebecabkerr is there top level component we can use for this quick sanity unit test?

File: README.md (hunk 3 above)
Comment (ethanhou99) [reply README.md]: Yes, I just modified this part with a simple summary.

File: README.md (hunk 2 above)
Comment (ethanhou99) [reply README.md]: Sure, the folder name was updated.

File: README.md [hunk 4]
```diff
@@ -62,6 +62,12 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
+## Unit Test
```
Comment (jcaianirh) [inline README.md]: npm test already has a section above. just modify that section to add this content to it.

Comment (ethanhou99) [reply README.md]: OK

**Comment by jcaianirh** [comment]
the test directory rename shouldn't be tests it should be __tests__

**Pull request merged**